### PR TITLE
[bitnami/ejbca] Updated README

### PR DESCRIPTION
**Pull request by vikram-bitnami**
**Description of the change**

Updated README

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**
This should not be landed until https://github.com/bitnami/charts-docs/pull/62 and https://github.com/bitnami/charts-docs/pull/61 are merged and the changes live on docs.bitnami.com

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/ejbca/README.md [hunk 1]
```diff
@@ -216,94 +216,55 @@ $ helm install my-release -f values.yaml bitnami/ejbca
...
 
-The `image` parameter allows specifying which image will be pulled for the chart.
+## Configure a private registry
 
-#### Private registry
+The `image.tag` parameter allows specifying which image will be pulled for the chart, as described in the [documentation for using a different image version](https://docs.bitnami.com/kubernetes/apps/ejbca/configuration/change-image-version/).
 
-If you configure the `image` value to one in a private registry, you will need to [specify an image pull secret](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod).
+When configuring the `image.tag` parameter to use an image in a private registry, it is necessary to also [specify an image pull secret](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod). This image pull secret must be passed to the chart at deployment time using the `imagePullSecrets` parameter in a `values.yaml` file (it cannot be passed using the `--set` parameter).
```
Comment (vikram-bitnami) [inline bitnami/ejbca/README.md:265]: @juan131 @javsalgar can we consider this paragraph to be applicable to all charts? If yes, should it be added to the common section at https://github.com/bitnami/charts-docs/blob/main/common/_change_image_version.md.erb?

File: bitnami/ejbca/README.md [hunk 2]
```diff
@@ -216,94 +216,55 @@ $ helm install my-release -f values.yaml bitnami/ejbca
...
 
 For example, for a release using `secretPassword` and `dbPassword` to scale up to a total of `2` replicas, the aforementioned parameters should hold these values `replicaCount=2`, `ejbcaAdminPassword=secretPassword`, `mariadb.auth.password=dbPassword`.
 
 > **Tip**: You can modify the file [values.yaml](values.yaml)
 
-### Sidecars
+### Configure Sidecars and Init Containers
 
-If you have a need for additional containers to run within the same pod as EJBCA (e.g. metrics or logging exporter), you can do so via the `sidecars` config parameter. Simply define your container according to the Kubernetes container spec.
+If additional containers are needed in the same pod as Jenkins (such as additional metrics or logging exporters), they can be defined using the `sidecars` parameter. Similarly, you can add extra init containers using the `initContainers` parameter.
```
Comment (juan131) [inline bitnami/ejbca/README.md:240]: Jenkins?

File: bitnami/ejbca/README.md (hunk 1 above)
Comment (juan131) [reply bitnami/ejbca/README.md:259]: Yes, this is generic. That said, the parameter is not `imagePullSecrets` but `global.imagePullSecrets` instead.

**Review by juan131 — CHANGES_REQUESTED** [review]
Minor changes requested.

**Review by juan131 — APPROVED** [review]
LGTM

**Pull request merged**